src/main/java/seedu/address/logic/LogicMeeting.java

```diff
@@ -0,0 +1,18 @@
+package seedu.address.logic;
+
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public interface LogicMeeting {
+    /**
+     * Executes the command and returns the result.
+     * @param commandText The command as entered by the user.
+     * @return the result of the command execution.
+     * @throws CommandException If an error occurs during command execution.
+     * @throws ParseException If an error occurs during parsing.
+     */
+    CommandResult execute(String commandText) throws CommandException, ParseException;
+
+    boolean isMeetingCommand(String commandText);
```

Comment:
Can remove all isMeetingCommand methods
Implemented only for LogicMode because need to know whether to switch mode in MainWindow, whereas it is certain that LogicMeeting will be handling a user input if the current mode is Meeting